invenio_communities/assets/semantic-ui/js/invenio_communities/members/members/manager_view/ManagerMembersResultItem.js

```diff
@@ -75,9 +75,9 @@ export class ManagerMembersResultItem extends Component {
                 key={result.id}
               >
                 {!result.is_current_user && (
-                  <SearchResultsRowCheckbox rowId={result.id} data={result} />
+                  <SearchResultsRowCheckbox className="mt-10" rowId={result.id} data={result} />
```

Comment:
could you provide a screenshot of how it works with more items on the list ?

Reply:
Sure, it would look like this
![image](https://user-images.githubusercontent.com/15194802/171159385-d7cfcbf9-51e7-4058-a878-f36b135ed686.png)

Reply:
I am asking in details, since the "flex" class should take care of the alignment, instead of adding margins to the image. Especially because the margin is not in relative units. I think the flex class is somehow broken or not applied properly, due to some changes. I would suggest investigating on that instead of applying the margin because margin is only a workaround in this case

Reply:
It looks good in all devices I tested, see:

## Ipad air
![127 0 0 1_5000_communities_123_members_q= l=list p=1 s=25 sort=name(iPad Air)](https://user-images.githubusercontent.com/15194802/171167505-68f147cb-fd17-49d9-90f5-05172869595d.png)

## Iphone SE
![127 0 0 1_5000_communities_123_members_q= l=list p=1 s=25 sort=name(iPhone SE)](https://user-images.githubusercontent.com/15194802/171167613-3a375f82-1a17-4d3c-a57a-f01133e55b3b.png)


But I can give it a try to get rid of the margins.

Reply:
After testing I am not sure we can get rif of the margins I applied to the logo the following rule to center it `align-self: center;` and remove the margins but the checkbox is still not well centered, see image:
![image](https://user-images.githubusercontent.com/15194802/171173362-97937335-9649-4992-91d5-c9fe72d3c5a7.png)

The checkbox has some margins inherited from [here](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/views/item.overrides#L123-L127) but the margin of 1em set there is too much. If I am not mistaken the grid size is static, personally I think setting the margins is OK in this case as it allows to align it correctly, see image:
![image](https://user-images.githubusercontent.com/15194802/171174265-72b0b2d2-415b-4d40-9a49-12f67e5063ee.png)